### PR TITLE
🌐 Add Chinese translation for `docs/zh/docs/history-design-future.md`

### PR DESCRIPTION
**Pull request by KAZAMA-DREAM**
🌐 Add Chinese translation for `docs/zh/docs/history-design-future.md`

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit 9a05717adf0d7e1582877d226bcba336d06f6b76 at: https://67589598.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 24e384d2de6696aed44697edd17f25accdcbb881 at: https://e364620c.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 9a05717adf0d7e1582877d226bcba336d06f6b76 at: https://141fd77a.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 24e384d2de6696aed44697edd17f25accdcbb881 at: https://f7d89d8f.fastapitiangolo.pages.dev

File: docs/zh/docs/history-design-future.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+<blockquote markdown="1">
+
+如果没有前人的工作，就不会有 **FastAPI**。
+
+之前许多已有的工具促使了它的诞生。
+
+几年来，我一直在避免创建一个新的框架。首先，我尝试使用许多不同的框架、插件和工具来解决 **FastAPI** 涵盖的所有功能。
+
+但在某种程度上，除了创建提供所有这些功能的东西，从以前的工具中汲取最好的想法，并使用以前甚至不可用的语言功能（Python 3.6+ 类型提示），以尽可能好的方式将它们组合在一起，别无选择。
```
Comment (Lenclove) [inline docs/zh/docs/history-design-future.md:27]: 这一行可以考虑加上“除了...之外...”的句式，感觉读起来更通顺。

**Comment by tiangolo** [comment]
📝 Docs preview for commit 9a05717adf0d7e1582877d226bcba336d06f6b76 at: https://a34a686f.fastapitiangolo.pages.dev

File: docs/zh/docs/history-design-future.md [hunk 2]
```diff
@@ -24,7 +24,7 @@
 
 几年来，我一直在避免创建一个新的框架。首先，我尝试使用许多不同的框架、插件和工具来解决 **FastAPI** 涵盖的所有功能。
 
-但在某种程度上，除了创建提供所有这些功能的东西，从以前的工具中汲取最好的想法，并使用以前甚至不可用的语言功能（Python 3.6+ 类型提示），以尽可能好的方式将它们组合在一起，别无选择。
```
Comment (KAZAMA-DREAM) [inline docs/zh/docs/history-design-future.md]: >

**Comment by tiangolo** [comment]
📝 Docs preview for commit bba4e5dace574471ce7ea0287e15d1f0d5f43d9e at: https://915ce69e.fastapitiangolo.pages.dev

File: docs/zh/docs/history-design-future.md (hunk 1 above)
Comment (KAZAMA-DREAM) [reply docs/zh/docs/history-design-future.md]: 好的，修改了一下

File: docs/zh/docs/history-design-future.md (hunk 2 above)
Comment (KAZAMA-DREAM) [inline docs/zh/docs/history-design-future.md]: > @Kludex

File: docs/zh/docs/history-design-future.md [hunk 3]
```diff
@@ -24,7 +24,7 @@
...
+但在某种程度上，除了创建提供所有这些功能之外，别无选择。通过从以前的工具中汲取最好的想法，并使用以前甚至不可用的语言功能（Python 3.6+ 类型提示），以尽可能好的方式将它们组合在一起。
```
Comment (KAZAMA-DREAM) [inline docs/zh/docs/history-design-future.md:27]: @Lenclove

Comment (KAZAMA-DREAM) [reply docs/zh/docs/history-design-future.md:27]: @Lenclove 有时间的话帮助我review一下这一个pr
#10400

**Comment by wdh99** [comment]
This file has already been translated. #3832

**Comment by tiangolo** [comment]
Thank you! It seems this was handled in another PR, so I'll pass on this one. But thanks for the effort! :coffee:

**Pull request closed without merging**